pkg/controller/releasemanager/incarnation.go

```diff
@@ -566,6 +566,25 @@ func (i *IncarnationCollection) releasable() []Incarnation {
 	return r
 }
 
+func (i *IncarnationCollection) unreleasable() []Incarnation {
+	releasableTags := []string{}
+	for _, incarnation := range i.releasable() {
+		releasableTags = append(releasableTags, incarnation.tag)
+	}
+
+	unreleasable := []Incarnation{}
+Outer:
```

Comment:
😻 learned about label breaks, nice

Reply:
usually means that you're doing somethign wrong